src/views/MyProfile/Components/VictoryPromiseDisplay/index.js

```diff
@@ -0,0 +1,340 @@
+      this.setState({
+        CC: 1,
+        LW: 1,
+        IM: 1,
+        LS: 1,
+        options: {
+          legend: {
+            display: false,
+          },
+          scale: {
+            gridLines: {
+              display: true,
+            },
+            ticks: {
+              display: false,
+              max: 1,
+              min: 0,
+              maxTicksLimit: 1,
+            },
+          },
+          tooltips: {
+            callbacks: {
+              label: function(tooltipItem, data) {
+                var label = data.labels[tooltipItem.index];
+                return label + ': 0';
+              },
+            },
+          },
+        },
+      });
+    }
+    this.setColor();
+  }
+
+  setColor() {
+    this.setState({
+      datasets: [
+        {
+          data: [this.state.CC, this.state.IM, this.state.LS, this.state.LW],
+          backgroundColor: [
+            this.state.CCColor,
+            this.state.IMColor,
+            this.state.LSColor,
+            this.state.LWColor,
+          ],
+        },
+      ],
+    });
+  }
+
+  render() {
+    let IMG_CC;
+    let IMG_IM;
+    let IMG_LS;
+    let IMG_LW;
+
+    if (this.state.IM_ON) {
+      IMG_CC = (
+        <img className="vpdesign" src={require('./images/On-CC.svg')} alt="Christian Character" />
+      );
+    } else {
+      IMG_CC = (
+        <img className="vpdesign" src={require('./images/Off-CC.svg')} alt="Christian Character" />
+      );
+    }
+
+    if (this.state.IM_ON) {
+      IMG_IM = (
+        <img className="vpdesign" src={require('./images/On-IM.svg')} alt="Intellectual Maturity" />
+      );
+    } else {
+      IMG_IM = (
+        <img
+          className="vpdesign"
+          src={require('./images/Off-IM.svg')}
+          alt="Intellectual Maturity"
+        />
+      );
+    }
+
+    if (this.state.LS_ON) {
+      IMG_LS = (
+        <img className="vpdesign" src={require('./images/On-LS.svg')} alt="Lives of Service" />
+      );
+    } else {
+      IMG_LS = (
+        <img className="vpdesign" src={require('./images/Off-LS.svg')} alt="Lives of Service" />
+      );
+    }
+
+    if (this.state.LW_ON) {
+      IMG_LW = (
+        <img className="vpdesign" src={require('./images/On-LW.svg')} alt="Leadership Worldwide" />
+      );
+    } else {
+      IMG_LW = (
+        <img className="vpdesign" src={require('./images/Off-LW.svg')} alt="Leadership Worldwide" />
+      );
+    }
+
+    const style = {
+      button: {
+        background: gordonColors.primary.cyan,
+        color: 'white',
+      },
+    };
+    const HoverText = withStyles(theme => ({
+      tooltip: {
+        backgroundColor: '#f5f5f9',
+        color: 'rgba(0, 0, 0, 0.87)',
+        maxWidth: 220,
+        fontSize: theme.typography.pxToRem(12),
+        border: '1px solid #dadde9',
+      },
+    }))(Tooltip);
+
+    let content;
+    //style={{backgroundColor: gordonColors.primary.blue}}
+
+    if (this.state.defaultVPMode) {
```

Comment:
Instead of duplicating the display in both content sections, could we keep only the parts that change in the content declaration, and write everything only once in the return statement?

Reply:
That's a good idea. Didn't think about that. Thank you!